### ISSUE TITLE
Feature Request: adding an option to apply transforms to each contour separately

### ISSUE THREAD

**Issue by aminabedi68** [fontforge/fontforge#4465]
the title is clear enough(i hope), I just want to say I think it's easy to implement(hope again!) and very useful for designers.

**Comment by skef** [comment]
FontForge already applies transforms only to selected (on-curve) points if any are selected. For your use case one can either select all the points in the contour(s) by hand or select one (on each) and run "Edit -> Select -> Points on Selected Contours", which has the shortcut Ctrl+Alt+, and then do the transform.

Is this sufficient?

**Comment by aminabedi68** [comment]
no, I mean fontforge can recognize contours from selected points, so it's possible to have an option to apply transforms individually(one after one for each contour), the difference is on the center of the transformation(on rotation, scaling,...) which makes a lot of works easier for users.

**Comment by skef** [comment]
So you're suggesting an option similar to "Origin: Center of Selection" but where the calculated center is specific to each contour. 

What is the use case? It's not obvious to me when someone would need that. (Or need it frequently enough to warrant a new mode.)
